string id
ServoVars[] servo_table
